dusk-abi/src/lib.rs

```diff
@@ -206,6 +208,14 @@ mod external {
         pub fn gas(value: i32);
 
         pub fn poseidon_hash(result: &mut u8, buffer: &u8, len: i32);
+        pub fn verify_proof(
+            pub_inp_len: i32,
```

Comment:
It's usually the opposite (and we kept that in the file): first you pass the starting point (e.g. the pointer), then the length.